.github/workflows/dockerpublish_restapi.yaml

```diff
@@ -2,18 +2,18 @@ name: Publish Docker image
 
 on:
   push:
-    # Publish `develop` as Docker `latest` image.
+    # Publish `vulnerability-restapi` as Docker `latest` image.
     branches:
-      - develop
+      - vulnerability-restapi
```

Comment:
Why the branch name is changed here? I guess this is done for testing? right?

Reply:
Yes, for testing purpose. Changed back now.